maro/rl/algorithms/dqn.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
```

Comment:
mv it to torch folder?

Reply:
I removed the torch folder since we are only supporting torch